main.rb

```diff
@@ -0,0 +1,29 @@
+require './app'
+
+app = App.new
+
+app.welcome
+
+loop do
+  app.menu
+  option = gets.chomp
+
+  case option
+  when '1'
+    app.list_books
+  when '2'
+    app.list_people
+  when '3'
+    app.input_person_info
+  when '4'
+    app.create_book
+  when '5'
+    app.create_rental
+  when '6'
+    app.list_rentals_for_person_id
+  when '7', 'q', 'quit'
+    break
+  end
+end
+
+puts 'Thank you for using this app!'
```

Comment:
- [ ] Kindly move this code for starting the app inside a main function and call it at the end of your file as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md#:~:text=this%20will%20be%20a%20method%20called%20main%20that%20is%20invoked%20at%20the%20end%20of%20your%20file.).